public/app.js

```diff
@@ -268,7 +268,13 @@ var app = new Vue({
     }
   },
   created: function() {
-    this.locale = window.localStorage.getItem('locale') || 'en';
+    this.locale = window.localStorage.getItem('locale') ||
```

Comment:
instead of setting this.locale, call setLocale instead.

That takes care of fetching the extra lang.json file for extra translations.